cdap-docs/developer-manual/source/metadata/metadata-ui.rst

```diff
@@ -22,33 +22,19 @@ Introduction
 The CDAP Metadata UI ("metadata management") lets you see how data is flowing into and out
 of datasets, streams, and stream views.
 
-It allows you to perform impact and root-cause analysis, delivers an audit-trail for
```

Comment:
There are still some references to Cask Tracker in this file. See lines 6-10, for instance.

Reply:
Not sure why it was added in the PR, but line 6 to 10 is not being used. Also verified that generated page from this file does not have any instances of 'cask tracker'